### DISCUSSION TITLE
By Difficulty filter depending on star rating in individual beat map difficulty

### DISCUSSION THREAD

**Discussion by leufix** [ppy/osu#23115] [Q&A] [answered]
will there be an update implementing this kind of difficulty filtering? Difficulty filter for individual beat map difficulties not depending on the highest difficulty of a particular song, like the Difficulty filter in lazer. or if it's actually in the game already and I just don't know how to do it? 

![image](https://user-images.githubusercontent.com/129760616/229522477-0e248fd3-bb5f-47fa-b5bb-65c5d207ab64.png)

**Comment by Rekunan** [comment, marked as answer]
It's planned yes #2547 #4919
